src/loadsave.cc

```diff
@@ -2234,11 +2267,17 @@ static void _ShowSlotList(int windowType)
     }
 
     // Pagination navigation
+    if (_currentSlotPage == 0) {
+        fontDrawText(gLoadSaveWindowBuffer + LS_WINDOW_WIDTH * (y + 0) + 95, "BACK", LS_WINDOW_WIDTH, LS_WINDOW_WIDTH, _colorTable[8804]);
```

Comment:
What about localization? Sfall had it's own ini file with translated strings.. But for CE I would think about adding entirely new MSG strings. That would mean editing msg files which are usually inside dat archives.. 

As a temporary solution, I suggest to introduce a new msg file like "game/ce_interface.msg" (don't worry about the 8 char limit, we're not in 1994 anymore). Add it to the repository as well. Then put all new strings for in-engine UI there.

Reply:
Sfall has a bunch of text messages related to it's features and tweaks, plus the Appearance Mod (the one that adds different races and hair styles) needs a few strings. Now they use an ini file, but this was done back when people were struggling to use what the engine has to offer. Nowadays sfall comes with it's own extra dat files which contains any new art and other resources for built-in features. CE could just follow this approach and a new MSG file as well.

Reply:
The problem with using vanilla msg files is they are already used by mods and localized. So better to have a new file for CE I think. Although if you pretend that no Fallout mods or localizations exist, then it's better to add strings to existing msg... But I think this is not a very useful approach, so long as CE is scoped as engine source port and not a complete game (with all resources included in repo).